### PR TITLE
Borderline-obnoxiously nitpicky style fixes

### PR DESCRIPTION
**Pull request by Turtlelord26**
Lots and lots of spaces in definition lists.

Hi #307!

### CODE REVIEW

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 1]
```diff
@@ -183,7 +184,7 @@ shield takes 30 damage and the plasma sword passes through the shield.
 MISCELLANEOUS THROWN WEAPONS
 ==============================
 
-    These weapons work similarly to guns. However your range is determined by 
+	These weapons work similarly to guns. However your range is determined by
```
Comment (bleehu) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt:187]: Why the double indent?

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 2]
```diff
@@ -1,28 +1,29 @@
...
 ===== UNARMED FIGHTING =====
 
 == Hand-to-Hand [untrained] Free ==
 
-Damage:    5 + STR + DEX
-Range:            Adjacent (1m)
-Accuracy Modifier:  -2
-DEX Requirement:    2
+Damage: 5 + STR + DEX
+Range: Adjacent (1m)
```
Comment (bleehu) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt:24]: I think removing the tabs to unalign the stats is going to make this harder to read in plaintext. Personally, I don't mind, but I'm trying to advocate for 1Source in his absence.

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt (hunk 1 above)
Comment (bleehu) [reply CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt:187]: ah. Our indent settings are different. We should probably standardize those.

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt [hunk 3]
```diff
@@ -1,28 +1,29 @@
...
 ==============================
-    These weapons/attacks use an 'Accuracy Modifier' instead of
+
+	These weapons/attacks use an 'Accuracy Modifier' instead of
 a 'Miss Chance'. This modifier is added to your combat dex modifier and
 and any relevant skill(s) to help you beat your opponent's guard DC.
 Melee weapons don't normally pierce armor, unless they state otherwise.
 
-If you have Hand-to-Hand trained, +10 damage for all melee weapons and 
+	If you have Hand-to-Hand trained, +10 damage for all melee weapons and
```
Comment (bleehu) [inline CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt:10]: good catch!

**Review by bleehu — APPROVED** [review]
shipit

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt (hunk 1 above)
Comment (Turtlelord26) [reply CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt:187]: I don't normally reject your indents and substitute my own but there were some of each in the same section so I made them all the same.

File: CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt (hunk 2 above)
Comment (Turtlelord26) [reply CharacterCreation/04_01_Melee_&_Thrown_Weapons.txt:24]: They weren't actually aligned or I wouldn't have attacked them. It just seemed random. (They were space characters, not tabs)

**Pull request merged**